### ISSUE TITLE
First draft for a Case Study on ORE

### ISSUE THREAD

**Issue by LinguList** [lexibank/lionnetyotonahua#8]
ORE offers the possibility to have "case studies", and for this dataset, we could make a case study that we call "Analyzing Lionnet's Reconstruction of Proto-Sonoran: A Computer-Assisted Case Study".

The content would be:

1. Introduction to the Dataset and Background -> @Maunus 
2. Background a) computer-assisted approaches (Mattis) b) debate about Proto-Soneram (@maunus)
3. Materals and Methods (mostly Mattis on the technical summary)
4. Analysis (here, one would point to the problem of spurious evidence, some general notes with statistics on the reconstruction, like correspondence pattern regularity, and then some conclusion, if the evidence is okay enough: Mattis does the statistics, and @maunus would derive conclusions from inspecting the data and the correspondences
5. Outlook

This is just one suggestion as of how we could proceed.

**Comment by Maunus** [comment]
This sounds good to me. It will be very light on debate and argumentation - I don't think we're likely to find anything that is really contradictory - since he isn't really making a particular argument that isn't already generally accepted. There may be some interesting things about borrowing between the branches obscuring the relation - even borrowing of a soundrule from Tepiman to Opata (y > d).

**Comment by LinguList** [comment]
Nice. Even if we only point to NUMBERS where before people would rather talk about examples, we be able to provide some added value, in so far as we can now say, we COUNT cases that conform to a given sound change / sound correspondence, while before it was more mentioned and assumed to be just true, without counting.

**Comment by Maunus** [comment]
That is a good point - we can perhaps identify some sound changes as better supported or clearer than others, and some as being tentative untill further examples are supplied. I am very frustrated with a lot of the changes where I can't identify the conditioning factor for example correspondences between r-y-s seem to almost be randomly distributed among the SUA languages.

**Comment by Maunus** [comment]
Also we can definitely point to the need to repeat with a larger database - Lionnet himself in the article points out the need to add Corachol and Nahuatl - and we can mention the usefulness of adding more on Tepiman - and then make the point that being interoperable the database is easy to incorporate in larger studies.

**Comment by LinguList** [comment]
Yes, perfect. That makes this first study something like an "ouverture" that leads to the big drama later :)

**Comment by xrotwang** [comment]
Sorry to just rush in here. The last point @Maunus mentioned got me interested: Is there a chance one could actually dig up the "missing data" somwhere and add it to a separate dataset - and then actually do the "let's merge and see" part in this study?

**Comment by Maunus** [comment]
That's basically the point. I have the missing data already (for Corachol-Nahua) - the data for Tepiman too is there but needs to be typed up.

**Comment by LinguList** [comment]
@xrotwang, I thought of making sure this first datasets works for now, but thinking about it now: making this about the merging is something we MISS so far in the work on CLDF.

**Comment by LinguList** [comment]
We have always assumed that merging is automatically done somewhere (in big databases), but not for these projects. So there is definitely a chance to expand this already now and see how the merging works.

**Comment by LinguList** [comment]
@Maunus, if you agree, we'd need another repository for the other data, or more specifically, one repository for each resource (if you have two comparative collections by two authors, it would be two more repositories). One question we may have problems with is concept coverage, as we cannot tell by now how many concepts overlap (but we can tell directly through Concepticon links, once we have the other data in CLDF).

**Comment by xrotwang** [comment]
One of the desiderata for CLDF always was to have enough and explicit information to merge data, giving preference to specified sources. And it looks like this could be a test case for this. (But I wouldn't want to push any work on you.)

**Comment by LinguList** [comment]
I'd say, we can proceed without having more work. As @Maunus anyway plans to have a larger DB where data are merged from different sources, this is a definite target, and if you, @xrotwang, have time to help a bit with the process later, this would also be excellent, as this would also allow for example to have a larger CLLD database online, etc. So we can just see where this goes. We can even just write the small case study on one dataset first and conclude that later this should be united with more resources.

**Comment by Maunus** [comment]
That is definitely my goal. So for Corachol and Nahua I haven't published the data yet, because I still have cognate sets I want to add, and sources to specify - but I hope to have it published somehow within the next year or so. I would want to merge this with this Lionnet database and perhaps also with Ken Hill and Wick Miller's cognate set database (there will be some repetition between them, but also a lot of non-shared content in each).

https://escholarship.org/uc/item/9px6p8h8

**Comment by Maunus** [comment]
Then eventually, my next project will be to make dictionaria format dictionaries of several varieties of Cora, Huichol and Nahuatl - which will make up an even larger interoperable dataset.

**Comment by LinguList** [comment]
The Ken Hill etc. Dataset looks also as if we can run it through our CLDF workflow, parsing data, etc.

**Comment by Maunus** [comment]
I know he has it in a digital database (I actually have a copy - but it uses a weird format called tswanalex or something like that). But maybe he'd give us access if we ask him.

**Comment by LinguList** [comment]
Good idea. If you could ask him. The data would be published in the CLDF 
format in his name, we would not steal anything, nor put any of our 
names on it, just help to lift it to another format and add more info to 
other datasets.
